packages/big-design/src/components/List/List.tsx

```diff
@@ -18,12 +17,12 @@ const StyleableList: React.FC<ListProps & PrivateProps> = ({ forwardedRef, updat
   const { height, width } = useWindowSize();
 
   useIsomorphicLayoutEffect(() => {
-    async function scheduleUpdate() {
-      await update();
+    function scheduleUpdate() {
+      update();
```

Comment:
I'm not very familiar with this function - why remove the `async` aspect of it?

Reply:
This may actually be incorrect. I have it typed incorrectly as a non `asynchronous` function, which is why I changed it to non async. However, it looks like it does return a `Promise`.

Reply:
The types were wrong in the previous downshift version, got confused by this a few days ago.